resources/xlf/en/sql.xlf

```diff
@@ -4750,6 +4750,9 @@ Error: {1}</source>
     <trans-unit id="accountDialog.didNotPickAuthProvider">
       <source xml:lang="en">You didn't select any authentication provider. Please try again.</source>
     </trans-unit>
+    <trans-unit id="accountDialog.loadingProviderLabel">
```

Comment:
did you manually add this? in ADS we only need to use localize(key, value) and the rest will be taken care by the localization process, why are we updating this in this PR?

Reply:
I ran the localization process on this Alan, just to save time, but I can remove it afterwards if needed.